test/performance/README.md

```diff
@@ -1,128 +1,168 @@
-## Stress Test Experiments
+## KS Performance Experiments
 
 
 ## Generate the workload traffic
 
-1. Clone the following fork of the clusterloader2 repo: 
+1. Clone the following clusterloader2 repo: 
 
-```bash
-$ git clone -b ks-provider https://github.com/dumb0002/perf-tests.git
-```
+   ```bash
+   $ git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
 
-2. Configure the parameters of your experiment:  
+   Set the variable `WORK_DIR` to the path of the directory where you cloned your repo:
```

Comment:
The command shown below is actually setting WORK_DIR to a _subdirectory_ of the cloned repo, right?

Reply:
Yes, and improved the writing to make that clear.